mava/wrappers/jumanji.py

```diff
@@ -236,16 +261,19 @@ def step(self, state: State, action: chex.Array) -> Tuple[State, TimeStep]:
             global_state=global_state,
             step_count=jnp.repeat(timestep.observation.step_count, self._env.num_agents),
         )
+        timestep.reward = jnp.sum(timestep.reward)
```

Comment:
I don't think this is correct, this makes it always a joint reward, what I think we need to do is create a `MARLRewardWrapper` that takes any environment with only 1 reward and repeats it `num_agents` number of times and then we must just handle this correctly in the loss functions and always wrap environments such as RWARE with this wrapper. 

@RuanJohn  and @OmaymaMahjoub your opinions are needed here

Reply:
This seems like a good idea, we can also use one wrapper for the shared reward and another for the opposite case.
And yes, in this case, we only used the shared reward and added it to get similar results to the benchmark paper where the sum of the individual rewards was used as the shared reward.